examples/gRPC/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.1)
+
+set(TESTING_CONFIG_OPT FILEPATH ${CMAKE_CURRENT_LIST_DIR}/config.cmake)
+include("../common.cmake")
+
+project(download-gRPC)
+
+hunter_add_package(gRPC)
+
+set(CMAKE_CXX_STANDARD_REQUIRED true)
```

Comment:
C++ standard and PIC is something global and should be moved to toolchain. You can set default toolchain like this:
* https://github.com/ethereum-mining/ethminer/blob/e27a96284645af2b5cd7d3e8e36bab34f1a6ab3f/CMakeLists.txt#L4